codespell_lib/tests/test_basic.py

```diff
@@ -250,7 +250,7 @@ def test_exclude_file(tmpdir, capsys):
     """Test exclude file functionality."""
     d = str(tmpdir)
     with open(op.join(d, 'bad.txt'), 'wb') as f:
-        f.write('1 abandonned 1\n2 abandonned 2\n'.encode('utf-8'))
+        f.write('1 abandonned 1\r\n2 abandonned 2\r\n'.encode('utf-8'))
```

Comment:
You're only checking for \r\n now, which means it would be possible for us to make a change which requires this exclusively. Can you make one of them \n and one \r\n to cover both cases?

Actually I guess for a good test here we should have four lines in bad.txt and four in tmp.txt so we can check all combinations work.